pkg/ambex/main.go

```diff
@@ -192,6 +196,13 @@ func parseArgs(ctx context.Context, rawArgs ...string) (*Args, error) {
 		dlog.Errorf(ctx, "Invalid AMBASSADOR_AMBEX_SNAPSHOT_COUNT: %s, using %d", numsnapStr, args.numsnaps)
 	}
 
+	// edsByPass will bypass using EDS and will insert the endpoints into the custer data manually
+	// This is a stop gap solution to resolve 503s on certification rotation
+	edsByPass := os.Getenv("AMBASSADOR_EDS_BY_PASS")
+	if edsByPass == "true" {
+		args.edsByPass = true
```

Comment:
I think we should add a log entry so that it is known that the feature flag is enabled

Reply:
added